editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -4459,9 +4461,6 @@ void SpatialEditor::_init_indicators() {
 
 		VisualServer::get_singleton()->instance_geometry_set_cast_shadows_setting(origin_instance, VS::SHADOW_CASTING_SETTING_OFF);
 
```

Comment:
Style issue with this empty line before the closing `}`.

Reply:
sorry about that, forgot to clang-format